<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameAdderDialog</class>
 <widget class="QDialog" name="GameAdderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Adder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Images/Leaner24.png</normaloff>:/Images/Leaner24.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>All Games</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="9,0,0">
         <item>
          <widget class="QTreeWidget" name="treeWidgetAllGames">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>100</number>
           </attribute>
           <column>
            <property name="text">
             <string>Game Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Gamerscore</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Award Count</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,5,1">
           <item>
            <widget class="QLabel" name="imgThumbnail">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,2">
             <item>
              <widget class="QLabel" name="lblGameName">
               <property name="text">
                <string/>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTitleID">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="lblGamerscore_2">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="Resources.qrc">:/Images/gamerscore.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblGamerscore">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="lblGamerscore_3">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="Resources.qrc">:/Images/achievement.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblAchievements">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="lblAvatarAwards_2">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="Resources.qrc">:/Images/avatar.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblAvatarAwards">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="txtSearch">
             <property name="placeholderText">
              <string>Search...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnShowAll">
             <property name="maximumSize">
              <size>
               <width>41</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/undo.png</normaloff>:/Images/undo.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnShowAll_2">
             <property name="text">
              <string>Awards Only</string>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/trophy.png</normaloff>:/Images/trophy.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>lblGameName</zorder>
      <zorder>lblAvatarAwards</zorder>
      <zorder>lblAvatarAwards_2</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Queue</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeWidget" name="treeWidgetQueue">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>100</number>
           </attribute>
           <column>
            <property name="text">
             <string>Game Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Gamerscore</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Award Count</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Add Games</string>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/add.png</normaloff>:/Images/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Cancel</string>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/cancel.png</normaloff>:/Images/cancel.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
